### PR TITLE
feat(WebexCreateSpace): emit operational metrics in create space component

### PR DESCRIPTION
**Pull request by sumit12690**
https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-326862

### CODE REVIEW

File: src/components/WebexCreateSpace/WebexCreateSpace.jsx [hunk 1]
```diff
@@ -62,17 +65,50 @@ export default function WebexCreateSpace({
...
   const onError = (error) => {
+    const endTime = window.performance.now();
+
+    emitMetrics({
+      fields: {
+        startTime,
+        endTime,
+        totalTime: endTime - startTime,
+      },
+      metricName: 'createSpaceTime',
```
Comment (kishorepmr) [inline src/components/WebexCreateSpace/WebexCreateSpace.jsx:76]: Can we have the name as createSpaceError and createSpaceSuccess?

**Review by kishorepmr — CHANGES_REQUESTED** [review]

**Comment by kishorepmr** [comment]
@sumit12690 General note no metric: there can already be a metric for create space emitting from anywhere so we might need to tag the client

File: src/components/WebexCreateSpace/WebexCreateSpace.jsx (hunk 1 above)
Comment (sumit12690) [reply src/components/WebexCreateSpace/WebexCreateSpace.jsx]: Done.

**Comment by sumit12690** [comment]
> @sumit12690 General note on metrics: there can already be a metric for create space emitting from anywhere(microservice or integration etc.,) so we might need to tag the client

@kishorepmr : I have added the tag key. Please check

**Review by kishorepmr — APPROVED** [review]

**Review by harsh789rai — APPROVED** [review]

**Pull request merged**